Plugins/IMA/IMAPlugin.swift

```diff
@@ -312,6 +312,7 @@ enum IMAState: Int, StateProtocol {
         case .STREAM_LOADED: self.notify(event: AdEvent.AdStreamLoaded())
         case .TAPPED: self.notify(event: AdEvent.AdTapped())
         case .THIRD_QUARTILE: self.notify(event: AdEvent.AdThirdQuartile())
+        default: break 
```

Comment:
lets add the missing unhandled events so we will know what is possible, using default is not the preferred way.